crates/router/src/connector/utils.rs

```diff
@@ -176,6 +179,13 @@ impl<Flow, Request, Response> RouterData for types::RouterData<Flow, Request, Re
             .and_then(|a| a.address.as_ref())
             .ok_or_else(missing_field_err("billing.address"))
     }
+
+    fn get_billing_address_with_phone_number(&self) -> Result<&api::Address, Error> {
```

Comment:
get_billing method is already present right, do we need this?

Reply:
We need this because get_billing_address() will return `AddressDetails` struct, we require `Address` struct